Framework/PythonInterface/plugins/algorithms/FindSXPeaksConvolve.py

```diff
@@ -109,16 +110,36 @@ def PyInit(self):
             validator=FloatBoundedValidator(lower=0.0),
             doc="Minimum peak size as a fraction of the kernel size.",
         )
+        self.declareProperty(
+            name="PeakFindingStrategy",
+            defaultValue="IOverSigma",
+            direction=Direction.Input,
+            validator=StringListValidator(["VarianceOverMean", "IOverSigma"]),
+            doc="PeakFindingStrategy=IOverSigma will find peaks by integrating data using a shoebox kernel and looking"
+            " for peaks with I/sigma > ThresholdIoverSigma. PeakFindingStrategy=VarianceOverMean will look for "
+            "peaks with local variance/mean > ThresholdVarianceOverMean.",
+        )
+        self.declareProperty(
+            name="ThresholdVarianceOverMean",
+            defaultValue=3.0,
+            direction=Direction.Input,
+            validator=FloatBoundedValidator(lower=1.0),
+            doc="Threshold value for variance/mean used to identify peaks.",
+        )
+        use_variance_over_mean = EnabledWhenProperty("PeakFindingStrategy", PropertyCriterion.IsNotDefault)
+        self.setPropertySettings("ThresholdVarianceOverMean", use_variance_over_mean)
```

Comment:
From a user perspective, isn't it better to keep `ThresholdIoverSigma` and  `ThresholdVarianceOverMean` variables closer and next to each other that becomes active/inactive based on the selection of `PeakFindingStratergy` after it?

![image](https://github.com/mantidproject/mantid/assets/142818102/250305f6-8d49-4c28-a4ee-9b84e9a86b94)

Reply:
yeah understood, I guess users won't bother much since there is only a small number of params to fill.